spec/lib/active_merchant/billing/affirm_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+RSpec.describe ActiveMerchant::Billing::Affirm do
+
+  let(:gateway_options) do
+    {
+      public_api_key: "XPSQ3CA7PLN7CJCK",
+      private_api_key: "w9mxkQUryKjTYDqOfSvYTeTGoLIURKpU",
+      test_mode: true
+    }
+  end
+
+  let(:affirm_payment_source) { create(:affirm_checkout, token: "MFLJS57G6BT5OQXN") }
+
+  subject do
+    ActiveMerchant::Billing::Affirm.new(gateway_options)
+  end
+
+  describe "#authorize" do
```

Comment:
We can just use vcr rspec metadata here instead of defining the names ourselves.